22-decay-tree-fitter.md

```diff
@@ -11,9 +11,9 @@ minutes: 10
 > * Apply a mass constraint
 > * Inspect the refitted decay tree
 
-Once you have made a hypothesis on the chain of decays that have lead to your final state, you then can incorporate the additional knowledge that comes with this hypothesis to get a new best estimate for the particle parameters - in particular their momenta. The additional knowledge is represented as constaints, which your decay tree has to fulfill.
+Once you have made a hypothesis on the chain of decays that have lead to your final state, you then can incorporate the additional knowledge that comes with this hypothesis to get a new best estimate for the particle parameters - in particular their momenta. The additional knowledge is represented as constraints, which your decay tree has to fulfill.
```

Comment:
“that have lead to your final state” → “that lead to your final state”.

“parameters - in” → “parameters — in” (sorry :smile: the former is a hyphen, the latter is an em-dash, which is more grammatically correct here)